packages/datadog-plugin-mysql/src/index.js

```diff
@@ -9,13 +9,13 @@ class MySQLPlugin extends DatabasePlugin {
   start ({ sql, conf: dbConfig }) {
     const service = getServiceName(this.config, dbConfig)
 
-    this.startSpan('mysql.query', {
+    this.startSpan(`${this.system}.query`, {
```

Comment:
With this change does it mean that mysql2 had a span called `mysql.query` which is now renamed to `mysql2.query`? And if so is that ok?

Reply:
The term `system` comes from [OpenTelemetry](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md#connection-level-attributes).